x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ServiceAccountAuthenticator.java

```diff
@@ -52,7 +49,7 @@ class ServiceAccountAuthenticator implements Authenticator {
         this.authenticationMetrics = new SecurityMetrics<>(
             SecurityMetricType.AUTHC_SERVICE_ACCOUNT,
             meterRegistry,
-            this::buildMetricAttributes,
+            serviceAccountToken -> Map.of(ATTRIBUTE_SERVICE_ACCOUNT_ID, serviceAccountToken.getAccountId().asPrincipal()),
```

Comment:
curious, what is the meaning of service_account_id in this context? I was a bit worried when seeing the name, but values look ok (`elastic/auto-ops`, `elastic/kibana`, ...).
Is `_id` the right suffix here? If easy to change (not used anywhere), could this be `service_account_type` or even just `service_name`. btw note, `.` is translated to `_` for attribute names. These labels might be easier to discover if using that representation.

Reply:
That's fine, keep as is 👍  I considered rejecting labels named `*_id`, but not sure that will help much to introduce other high cardinality labels.